src/crimson/net/ProtocolV2.cc

```diff
@@ -95,27 +95,23 @@ inline uint64_t generate_client_cookie() {
 
 } // namespace anonymous
 
-namespace fmt {
 template <>
-struct formatter<seastar::lowres_system_clock::time_point> {
+struct fmt::formatter<seastar::lowres_system_clock::time_point> {
   // ignore the format string
   template <typename ParseContext>
   constexpr auto parse(ParseContext &ctx) { return ctx.begin(); }
 
   template <typename FormatContext>
   auto format(const seastar::lowres_system_clock::time_point& t,
 	      FormatContext& ctx) {
-    struct tm bdt;
-    time_t tt = std::chrono::duration_cast<std::chrono::seconds>(
+    std::time_t tt = std::chrono::duration_cast<std::chrono::seconds>(
       t.time_since_epoch()).count();
-    localtime_r(&tt, &bdt);
     auto milliseconds = (t.time_since_epoch() %
```

Comment:
(Not a problem with the change, but with the existing code) 'milliseconds' is a std::chrono much-used name, and the variable name may cause confusion.

Reply:
i think it's safe and not that bad. as `std::chrono` also has `seconds` and `hours` defined in it. and in C++20, we will have `std::chrono::day`. these are all good variable names we will be missing otherwise.